src/lib_ccx/ccx_encoders_smptett.c

```diff
@@ -213,12 +216,15 @@ int write_cc_buffer_as_smptett(struct eia608_screen *data, struct encoder_ctx *c
 			// COLUMNS is actually 90% of the screen size
 			// Add +10% because column 0 is at position 10%
 			col1 = ((100 * firstcol) / (COLUMNS / 0.8)) + 10;
+			col1_int = col1;
```

Comment:
This is going to trigger a compiler warning about loss of precision. i.e. use casting.